external-docs/docs/guides/shape-data.md

```diff
@@ -1,3 +1,241 @@
 # Shaping your data
 
-todo: Explain the shapes of data we accept. Issue #17.
+> This page discusses the shapes of input data supported by csvcubed with the aim to help you understand which to use and how to generate data in the right shape.  
+
+csvcubed requires that all CSV data inputs are provided in one of two specialised form of [tidy data](../glossary/index.md#tidy-data):
+
+* the [canonical approach](#canonical-shape) - the **default recommended shape** accepted by csvcubed. It is the **most flexible** but also the most **verbose**.
+* the [pivoted approach](#pivoted-shape) - a **terser** shape currently only compatible with data sets containing a **single measure**.
```

Comment:
"data sets" should be "datasets".

Reply:
I agree with you, however the [Government Digital Service guidelines](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#data-set) do not. We're following them.